packages/go_router/doc/type-safe-routes.md

```diff
@@ -0,0 +1,73 @@
+Instead of using URL strings to navigate, go_router supports
+type-safe routes using the go_router_builder package. 
+
+To get started, add go_router_builder, build_runner, and build_verify to the
```

Comment:
Should we make `go_router_builder`, `build_runner`, and `build_verify` links? Users may want to learn more about what they're installing.